upf/upf_pfcp_server.c

```diff
@@ -1139,6 +1147,11 @@ void upf_server_send_heartbeat (u32 node_idx)
   SET_BIT (req->grp.fields, HEARTBEAT_REQUEST_RECOVERY_TIME_STAMP);
   req->recovery_time_stamp = psm->start_time;
 
+  upf_pfcp_associnfo
```

Comment:
Aint this going to happen regardlessly if association is suspicious or not? It seems it will be happen for every normal heartbeat.

Reply:
Yeah, you're right. It will only be suspicious when sent by a heartbeat timeout. So maybe it would be better to log it in that case but not in send_heartbeat?